ydb/_topic_common/common.py

```diff
@@ -29,6 +30,12 @@ def wrapper(rpc_state, response_pb, driver=None):
     return wrapper
 
 
+def wrap_create_asyncio_task(func: typing.Callable, task_name: str, *args, **kwargs):
+    if sys.hexversion < 0x03080000:
+        return asyncio.create_task(func(*args, **kwargs))
+    return asyncio.create_task(func(*args, **kwargs), name=task_name)
```

Comment:
It seems weird how we pass name and args. This wrapper could get already created task and use `task.set_name(name)` in case of python 3.8+. In this case you will don't have any problems between `asyncio.create_task()` and `loop.create_task()` because they both return a Task you can pass into your wrapper

Also this wrapper could have a better name, because the current one doesn't explain anything we do with the task.

I don't have anything against hexversion, but I'm sure we can omit this check on every wrapper call. We could use this split during function declaration stage. In this case, if we will have only one `if` call, think about switching to `version_info` for better readability (i'm not insist here)